src/test/java/fruite/store/service/impl/FileWriterServiceImplTest.java

```diff
@@ -0,0 +1,54 @@
+package fruite.store.service.impl;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class FileWriterServiceImplTest {
+    private static final String VALID_FILE_NAME_AND_PATH =
+            "src/test/resources/report-by-day.csv";
+    private static final String INVALID_FILE_NAME_OR_PATH
+            = "src/tst/resorces/report-by-day.csv";
+    private FileWriterServiceImpl fileWriterService = new FileWriterServiceImpl();
+
+    @AfterClass
+    public static void afterClass() {
+        try {
+            Files.deleteIfExists(Path.of(VALID_FILE_NAME_AND_PATH));
+        } catch (IOException e) {
+            throw new RuntimeException("Can't correctly clear result files after test: ", e);
+        }
+    }
+
+    @Test
+    public void writeToFile_validPathOrFileName_ok() {

```

Comment:
You can compare with `String `  using readString to get file data ( to simplify this)